src/main/resources/liquibase/crop_changelog/23_4_0.xml

```diff
@@ -25,4 +25,464 @@
 		<comment>Create json_props column in phenotype_aud table</comment>
 		<addColumn tableName="phenotype_aud"><column name="json_props" type="CLOB"></column></addColumn>
 	</changeSet>
+	
```

Comment:
Please remove the changes here in 23_4_0.xml.

Reply:
done!